docs/artifacts/five-clauses-zero-mechanisms.html

```
@@ -0,0 +1,525 @@
+<pre class="mermaid">
+%%{init: {'theme':'base','themeVariables':{'background':'#FCFCFE','primaryColor':'#F2F1F7','primaryTextColor':'#16141F','primaryBorderColor':'#5B3FD6','lineColor':'#565073','fontFamily':'IBM Plex Mono, monospace','fontSize':'13px'}}}%%
+flowchart LR
+  L0["0 · mandate into<br/>CLAUDE.md + AGENTS.md"]
+  L5a["5a · read-only plugin<br/>+ kb-serve in dotfiles"]
+  L1a["1a · tool-set lift<br/><i>pure refactor</i>"]
+  L3["3 · URL index<br/><i>wired to no hook</i>"]
+  L2["2 · kb-add as a module"]
+  L3b["3b · reconciliation<br/>detector"]
+  L1b["1b · wire the guard"]
+
+  L0 --> L1a
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)